etcdutl/snapshot/v3_snapshot.go

```diff
@@ -384,6 +384,8 @@ func (s *v3Manager) copyAndVerifyDB() error {
 
 	// db hash is OK, can now modify DB so it can be part of a new cluster
 	db.Close()
+	dbClosed = true
```

Comment:
probably we can just remove line 386 (`db.Close()`) and `dbClosed` (line 343)

Reply:
yes, it's easier that way, updated.